autonomy/cli/utils/click_utils.py

```diff
@@ -107,6 +107,4 @@ def convert(
     ) -> Optional[Path]:
         """Convert path string to `pathlib.Path`"""
         path_string = super().convert(value, param, ctx)
-        if path_string is None:
-            return path_string
-        return Path(path_string)
+        return path_string and Path(path_string)
```

Comment:
How’s this identical?

Reply:
it is not, if you run `autonomy analyse fsm-specs --package ""` it will actually return an empty string instead of a Path